lib/tasks/demo.rake

```diff
@@ -1,6 +1,42 @@
 # frozen_string_literal: true
 
 namespace :demo do
+desc 'Backup the database'
+  task backup_database: :environment do
+    raise 'This task is only for use in a development environment' unless Rails.env == 'development' || ENV['DISABLE_DATABASE_ENVIRONMENT_CHECK']
+    username = ActiveRecord::Base.configurations.configurations[1].config['username']
+    database = ActiveRecord::Base.configurations.configurations[1].config['database']
+    system "mysqldump --opt --user=#{username} #{database} > sara_database_#{Time.now.to_i}.sql"
+  end
+
+  desc 'Restore the database'
+  task restore_database: :environment do
+    # Example usage: rake demo:restore_database FILE=sara_database_1606835867.sql
+    raise 'This task is only for use in a development environment' unless Rails.env == 'development' || ENV['DISABLE_DATABASE_ENVIRONMENT_CHECK']
+    raise 'FILE environment variable must be set to run this task' if ENV['FILE'].nil?
+    username = ActiveRecord::Base.configurations.configurations[1].config['username']
+    database = ActiveRecord::Base.configurations.configurations[1].config['database']
+    system "mysql --user=#{username} #{database} < #{ENV['FILE']}"
+  end
+
+  # Duplicate existing monitoree data
+  # Note: comment out `around_save :inform_responder, if: :responder_id_changed?` in app/models/patient.rb to speed up process
+  desc 'Generate N many more monitorees based on existing data'
+  task create_bulk_data: :environment do
```

Comment:
When running this task as `COUNT=1000 bundle exec rails demo:create_bulk_data` I get this error.
```
#<Thread:0x00007ff08107c2d0@/Users/jkufro/Desktop/SaraAlert/code/SaraAlert/lib/tasks/demo.rake:32 run> terminated with exception (report_on_exception is true):
Traceback (most recent call last):
	11: from /Users/jkufro/Desktop/SaraAlert/code/SaraAlert/lib/tasks/demo.rake:32:in `block (4 levels) in <main>'
	10: from /Users/jkufro/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/activesupport-6.0.3.4/lib/active_support/core_ext/range/each.rb:9:in `each'
	 9: from /Users/jkufro/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/activesupport-6.0.3.4/lib/active_support/core_ext/range/each.rb:9:in `each'
	 8: from /Users/jkufro/Desktop/SaraAlert/code/SaraAlert/lib/tasks/demo.rake:32:in `block (5 levels) in <main>'
	 7: from /Users/jkufro/Desktop/SaraAlert/code/SaraAlert/lib/tasks/demo.rake:919:in `deep_duplicate_patient'
	 6: from /Users/jkufro/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/activerecord-6.0.3.4/lib/active_record/relation/delegation.rb:87:in `each'
	 5: from /Users/jkufro/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/activerecord-6.0.3.4/lib/active_record/relation/delegation.rb:87:in `each'
	 4: from /Users/jkufro/Desktop/SaraAlert/code/SaraAlert/lib/tasks/demo.rake:921:in `block in deep_duplicate_patient'
	 3: from /Users/jkufro/Desktop/SaraAlert/code/SaraAlert/lib/tasks/demo.rake:924:in `deep_duplicate_patient'
	 2: from /Users/jkufro/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/activerecord-6.0.3.4/lib/active_record/relation/delegation.rb:87:in `each'
	 1: from /Users/jkufro/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/activerecord-6.0.3.4/lib/active_record/relation/delegation.rb:87:in `each'
/Users/jkufro/Desktop/SaraAlert/code/SaraAlert/lib/tasks/demo.rake:929:in `block in deep_duplicate_patient': undefined method `save' for nil:NilClass (NoMethodError)
```